### PR TITLE
Web: minor UI styling issues

### PR DESCRIPTION
**Pull request by kimlisa**
- removes extra icon from role request info banner
- removes the `new` badge for aws external audit storage tile

### CODE REVIEW

File: web/packages/teleport/src/Integrations/Enroll/IntegrationTiles.tsx [hunk 1]
```diff
@@ -38,7 +38,8 @@ export function IntegrationTiles({
   hasIntegrationAccess?: boolean;
   hasExternalAuditStorage?: boolean;
 }) {
-  const externalAuditStorageEnabled = cfg.externalAuditStorage;
+  const externalAuditStorageEnabled =
+    cfg.entitlements.ExternalAuditStorage.enabled;
```
Comment (kimlisa) [inline web/packages/teleport/src/Integrations/Enroll/IntegrationTiles.tsx:42]: @mcbattirola just wanted to double check this change is equivalent and is ok

Comment (kimlisa) [reply web/packages/teleport/src/Integrations/Enroll/IntegrationTiles.tsx:42]: should `cfg.externalAuditStorage` then be marked deprecated??

Comment (mcbattirola) [reply web/packages/teleport/src/Integrations/Enroll/IntegrationTiles.tsx:42]: Yes, `cfg.externalAuditStorage` is the old way, we should use the entitlements now.

Here's where we set `cfg.externalAuditStorage` based on the entitlement value for compatibility: https://github.com/gravitational/teleport/blob/ee1726ef684b7162de241ba63674cfdcdbfe074b/lib/web/apiserver.go#L1872


> should cfg.externalAuditStorage then be marked deprecated??

Yes. Do you want me to open a PR, or you prefer including it in this one?

**Review by ravicious — APPROVED** [review]

**Review by r0mant — APPROVED** [review]

File: web/packages/teleport/src/Integrations/Enroll/IntegrationTiles.tsx (hunk 1 above)
Comment (kimlisa) [reply web/packages/teleport/src/Integrations/Enroll/IntegrationTiles.tsx:42]: i'll do it in another PR

**Pull request merged**